eng/common/TestResources/New-TestResources.ps1

```diff
@@ -92,6 +92,9 @@ param (
     [Parameter()]
     [switch] $SuppressVsoCommands = ($null -eq $env:SYSTEM_TEAMPROJECTID),
 
+    [Parameter()]
+    [switch] $UseUserCredentials,
```

Comment:
should this default to true if `$CI` is false?

Reply:
I'd rather pilot this change first because I think there will be downstream changes that are needed in test-resources files for many libraries.

Reply:
For .NET we can pass it as true from the test framework when bootstrapping resources.

Reply:
Also, I don't think we can just default to true as community contributors use this. We would probably need to default to true only if the subscription is a Microsoft subscription.